astilectron.go

```diff
@@ -67,14 +67,16 @@ type Astilectron struct {
 
 // Options represents Astilectron options
 type Options struct {
-	AcceptTCPTimeout   time.Duration
-	AppName            string
-	AppIconDarwinPath  string // Darwin systems requires a specific .icns file
-	AppIconDefaultPath string
-	BaseDirectoryPath  string
-	DataDirectoryPath  string
-	ElectronSwitches   []string
-	SingleInstance     bool
+	AcceptTCPTimeout     time.Duration
+	AppName              string
+	AppIconDarwinPath    string // Darwin systems requires a specific .icns file
+	AppIconDefaultPath   string
+	BaseDirectoryPath    string
+	DataDirectoryPath    string
+	ElectronSwitches     []string
+	SingleInstance       bool
+	SkipAstilectronSetup bool   // If true, the user must handle provisioning and executing astilectron.
```

Comment:
Could you rename it to `SkipAstilectronSetup`?

Reply:
What should I rename it to?

Reply:
Oops sorry, could you rename it to `SkipSetup`?

Reply:
Yep, done.